package.json

```diff
@@ -20,6 +20,7 @@
   "scripts": {
     "dev": "next dev",
     "build": "next build",
+    "build:netlify": "rm -rf .next/cache && next build",
```

Comment:
[Nextjs DataCache](https://nextjs.org/docs/app/building-your-application/caching#data-cache) is persistent between builds through the `.next/cache` and Netlify plugin. That is why between builds we were not seeing updates to the content. This little hack removes the cached files persistent between deployments.